tests/test_users.py

```diff
@@ -3,40 +3,98 @@
 from utils import MongoDb
 
 
-@pytest.fixture
-def get_connection():
-    col = MongoDb(host='localhost')
-    col.create_connection()
-    return col
-
-
 users_host = 'http://localhost:81'
 
+@pytest.fixture
+def collection():
+    con = MongoDb(host='localhost')
+    con.create_connection()
+    con.set_collection('users')
+    yield con 
+    con.drop()
 
-# create a using using the db
-# call users service and get the data
-# check that the data is as espected
-# delete the user using the database
-def test_get_user(get_connection):
-    get_connection.upsert('users', 100,
-                          {'name': 'John', 'email': 'test@email.eu'})
+def test_get_user(collection):
+    collection.upsert(
+        100,
+        {
+            'name': 'John', 
+            'email': 'test@email.eu'
+        }
+    )
     response = requests.get(url='{0}/users/100'.format(users_host)).json()
     assert response['_id'] == 100
     assert response['email'] == 'test@email.eu'
     assert response['name'] == 'John'
-    get_connection.delete('users', 100)
 
+def test_create_user(collection):
+    requests.put(
+        url='{0}/users/101'.format(users_host),
+        data={
```

Comment:
Please use the same formatting for accolades everywhere (https://en.wikipedia.org/wiki/Indentation_style)

Reply:
I am sorry what are accolades? I wanted to distinguish the arguments from the rest.
Do you want me to use this style?
```python
foo = long_function_name(var_one, var_two,
                         var_three, var_four)
```

Reply:
Sorry, curly brackets {}
Let's use the same style for all python files it's a best practice. I've mixed styles too unfortunately. sometimes i've written one liners other times a mix between other styles. Pick one and we'll format the whole project it's the best think to do..consistency. 
It's great that we collaborate, hidden things are brought to surface

Reply:
In that case we could use (https://github.com/psf/black). If you think this is good, I will set a Github action to format on every pull request and push.

Reply:
Hmm interesting, but isn't a bit brutal to reformat our code on every commit ? maybe send a warning ? or fail the push ?

Reply:
It can take our our mind off from formatting altogether. It's like what they claim
> You will save time and mental energy for more important matters
and I see lot of Github projects are using Black.

Reply:
Ok let's try it, we'll learn something. And if it's not suited for us will decide later

Reply:
got it. I personally use yapf and I've created a repo template for my personal projects. We could use pre-commit config to send a warning or fail to push using flake8 or pep8 guidelines. Please take a look at
https://github.com/neelabalan/pyproject-template

If you think black is little overloaded we can choose some other approach for failing push or sending a warning using pre-commit. I will raise a seperate issue for black code formatting where we can discuss about this